tests/handlers/aws/test_outputs.py

```diff
@@ -0,0 +1,135 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+
+import os
+from string import Template
+from typing import Any
+from unittest import TestCase
+
+import boto3
+import mock
+import pytest
+from botocore.client import BaseClient as BotoBaseClient
+from testcontainers.localstack import LocalStackContainer  # type: ignore
+
+from main_aws import handler
+from tests.handlers.aws.test_handler import ContextMock
+from tests.handlers.aws.utils import (
+    _class_based_id,
+    _load_file_fixture,
+    _logs_create_cloudwatch_logs_group,
+    _logs_create_cloudwatch_logs_stream,
+    _logs_retrieve_event_from_cloudwatch_logs,
+    _logs_upload_event_to_cloudwatch_logs,
+    _s3_upload_content_to_bucket,
+)
+from tests.testcontainers.logstash import LogstashContainer
+
+
+@pytest.mark.integration
+class TestLambdaHandlerLogstashOutputSuccess(TestCase):
+    def setUp(self) -> None:
+        lst = LocalStackContainer(image="localstack/localstack:1.1.0")
+        lst.with_env("EAGER_SERVICE_LOADING", "1")
+        lst.with_services("s3", "logs", "sqs")
+        self.localstack = lst.start()
+
+        aws_default_region = "us-east-1"
+        session = boto3.Session(region_name=aws_default_region)
+        self.s3_client = session.client("s3", endpoint_url=self.localstack.get_url())
+        self.logs_client = session.client("logs", endpoint_url=self.localstack.get_url())
+        self.sqs_client = session.client("sqs", endpoint_url=self.localstack.get_url())
+
+        self.logstash_http_port = 5043
+        lgc = LogstashContainer(port=self.logstash_http_port)
+        logstash_config = """\
```

Comment:
minor: over the codebase I think we tend to use python 3.6 f-strings rather than format()